cursos/kubernetes/03_configuracion/09_practica_guiada_HPA.md

```diff
@@ -12,7 +12,7 @@ kubectl create namespace hpa
 
 ```
 
-- Agora imos despregar un deployment que teña unha aplicación de emprego intensivo de cpu
+- Agora imos despregar un deployment que teña unha aplicación de emprego intensivo de cpu onde definimos os recursos da cpu
```

Comment:
This explanation should be more extensive: "Para que poida facer a media, compre definir unha sección resources.requests para saber cal e o consumo esperado de CPU / Memoria. De non ter estes valores o hpa non poderá facer os cálculos"

Reply:
I'll add that explanation